### PR TITLE
use a condafied version of the container

### PR DESCRIPTION
**Pull request by bgruening**
xref: https://github.com/hexylena/docker-rstudio-notebook/pull/36

### CODE REVIEW

File: tools/interactive/interactivetool_rstudio.xml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <tool id="interactive_tool_rstudio" tool_type="interactive" name="RStudio" version="0.1">
```
Comment (mvdbeek) [inline tools/interactive/interactivetool_rstudio.xml]:
```suggestion
<tool id="interactive_tool_rstudio" tool_type="interactive" name="RStudio" version="0.2">
```

**Comment by mvdbeek** [comment]
If this works we should create an exception for ITs and apply this directly to the 21.05 branch of the main Galaxy repo.

**Comment by bgruening** [comment]
I, or the new base image, broke nginx. So I needed to bypass nginx to make it work for now until I get help from @hexylena.

The image should work you can test it on usegalaxy.eu.

**Comment by bgruening** [comment]
xref: https://help.galaxyproject.org/t/the-rstudio-on-galaxy-i-cannot-install-the-seurat-package/6364/6

**Pull request merged**